design/ceph/ceph-zone.md

```diff
@@ -0,0 +1,183 @@
+# Rook Ceph Zone
+
+## Prerequisites
+
+A Rook Ceph cluster. Ideally a ceph-realm and a ceph-zone-group resource would have been started up already.
+
+## Ceph Multisite Overview
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
```

Comment:
nit: Instead of repeating this section in every doc, how about a link back to the main multisite doc?